src/txn/txn_recover.c

```diff
@@ -929,7 +929,7 @@ __hs_exists_local(WT_SESSION_IMPL *session, WT_CURSOR *metac, const char *cfg[],
  *     Run recovery.
  */
```

Comment:
Can you please add a comment noting that this function must not be called in Disagg mode?

Reply:
done!